### DISCUSSION TITLE
404 error on page refresh

### DISCUSSION THREAD

**Discussion by SID-dev-droid** [vercel/community#2382] [Help]
### Summary

Hi,

my application is working perfectly however, on page reload my chat page (/chat) I get a 404 error that the page is not found even though I was able to see it before. I believe I need to include a route or rewrite in my Vercel.config but from the configurations I have tried I don't think I am getting it right. Please let me know if you think you can help. I am running my front end out of the client folder, here is a image of my project's file structure and vercel.config.

### Additional information

![image](https://github.com/vercel/community/assets/104534833/65b116cb-1963-4701-97f5-2b1d7f0ce3b4)
![image](https://github.com/vercel/community/assets/104534833/5467f489-6252-4d2f-be5e-82bef8323c9b)
![image](https://github.com/vercel/community/assets/104534833/14e17ccb-1fb9-4293-bd2f-5a873d38bf7f)

**Comment by Valenmetal** [comment]
Hi! I have the same problem, f5 on any working page and i get a 404. Did you find the solution?
